docs/extras/advanced-targeting/self-targeting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Simple Self-Targeting
```

Comment:
```suggestion
title: Self-Targeting
```

Reply:
hi @andrewdmaclean! overall, i like the edits/organization! 

if we want to keep this feature within targeting i think that this is a good place for it but would stay away from calling it "simple" bc it's a bit counterintuitive to have it under a section titled "advanced". 

the reason i wanted a QA section would be to make sure teams understand the primary use-case for being able to force themselves into a variation. in addition we could add a similar call out to what you added guides ppl to use reusable audiences as well for QA/internal users! lmk what you think :)